frontend/src/components/module-viewer/LessonCompleted.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext } from "react";
+import { useHistory, useParams } from "react-router-dom";
+import { Button, Flex, Heading, Text, VStack } from "@chakra-ui/react";
+
+import { ModuleEditorParams } from "../../types/ModuleEditorTypes";
+import EditorContext from "../../contexts/ModuleEditorContext";
+import { HOME_PAGE } from "../../constants/Routes";
+import Banner from "../learner/Banner";
+
+const LessonCompleted = () => {
```

Comment:
Is this component meant to show at the end of a completed lesson? I don't think we have a need for that. The only completion page is at the end of a module. Maybe I'm misunderstanding something

Reply:
Also fix linter warning :)